ThirdPartyNoticeText.txt

```diff
@@ -162,7 +162,7 @@ For purposes of this definition, the normative portions of the Specification sha
 10.7.8. shall not be considered accepted by an implementer who manifests an intent not to accept the terms of the W3C Community RF Licensing Requirements license as offered by the licensor.
 10.7.9. The RF license conforming to the requirements in this policy shall be made available by the licensor as long as the Specification is in effect. The term of such license shall be for the life of the patents in question.
 I am encouraged to provide a contact from which licensing information can be obtained and other relevant licensing information. Any such information will be made publicly available. 
-10.8. You or Your. �You,� �you,� or �your� means any person or entity who exercises copyright or patent rights granted under this Agreement, and any person that person or entity controls.
+10.8. You or Your. “You,” “you,” or “your” means any person or entity who exercises copyright or patent rights granted under this Agreement, and any person that person or entity controls.

```

Comment:
I'm not sure this is an improvement... :wink:

Reply:
Okay, I think GitHub's diff display is just bugged.  It looks fine in my review comment, but under the Files Changed tab all the non-ASCII characters are mangled, whereas the original looks normal.  Does this file need a BOM or something?

Reply:
If that's what the license includes, that's what we include. I'm definitely not going to start arbitrarily replacing characters in someone else's license!

Reply:
See my second comment.  I wasn't implying you should edit out the smartquotes.

Reply:
I think this file was so broken before that nothing could tell what it was, hence fixing the encoding to UTF-8 without BOM.

Reply:
> See my second comment. I wasn't implying you should edit out the smartquotes.

Hah, yeah, I didn't realize that the other diff view got this wrong.